OpenRA.Mods.Common/SpriteLoaders/ShpTSLoader.cs

```diff
@@ -118,7 +118,9 @@ bool IsShpTS(Stream s)
 			while (w == 0 && h == 0 && f++ < imageCount);
 
 			s.Position = start;
-			return type < 4;
+
+			// Type 18 is a special case for RA2 low bridges.
```

Comment:
As far as I see it that parser doesn't have the "Check the size and format flag" our loader has. I didn't know that the type means compression type. For the first 3 frames of a RA2 low bridge tile I debugged the following

```
lobrdb01.sno
w 0 h 0 type 4
w 0 h 0 type 0
w 0 h 64256 type 18
```

with actual data being in the 2nd frame so it seems to be type 0 (uncompressed data) with no width/height which we discard as invalid. The 3rd frame has a gigantic height and my so called type 18 so I guess other data is stored in there.

Can I just remove the `w == 0 && h == 0` check or can you maybe explain for which files you added it so we can change this exceptional case?